dateparser/date_parser.py

```diff
@@ -32,7 +32,7 @@ def parse(self, date_string, parse_method, settings=None):
         else:
             if "local" in _settings_tz:
                 stz = get_localzone()
-                if hasattr(stz, "localize") and sys.version_info < (3, 6):
+                if hasattr(stz, "localize") and sys.version_info < (3, 9):
```

Comment:
Is this correct?

Reply:
Actually this is always False now, so this whole block can be removed. There may be other such places in the code so I suggest looking for them and cleaning them up, as a separate PR.